nixos/tests/networking.nix

```diff
@@ -514,12 +514,14 @@ let
                   local = "192.168.2.1";
                   remote = "192.168.2.2";
                   dev = "eth2";
+                  ttl = 225;
```

Comment:
just checked: `ip tunnel` reports the ttl of a tunnel, so we can check its output for the configured value. (optionally `ip -j tunnel` and parsing the json in python to avoid string comparisons for extra niceness)

Reply:
Done.

Reply:
Is there anything needed to be done by me to get this merged?

Reply:
no, just a grace period for others to provide comments if they're unhappy with anything.